File: client/src/app/shared/models/post.ts
export interface IPost {
  pageSize: number;
  pageIndex: number;
  count: number;
  data: Datum[];
}

export interface Datum {
  account: Account;
  accountId: number;
  title: string;
  description: string;
  createdDate: string;
  isDeleted: boolean;
  comments: Comment[];
  likes: Like[];
  id: number;
}

export interface Account {
  userName: string;
  email: string;
  password: string;
  accountType: number;
  createdDateTime: string;
  lastLoginDateTime: string;
  isActive: boolean;
  posts: null[];
  comments: Comment[];
  likes: Like[];
 
}

export interface Like {
  postId: number;
  likedbyAccountId: number;
  createdDate: string;
  isLiked: boolean;
  id: number;
}

export interface Comment {
  postId: number;
  accountId: number;
  description: string;
  createdDate: string;
  isDeleted: boolean;
  id: number;
}

File: Core/Entities/Post.cs
using System;
using System.Collections.Generic;

namespace Core.Entities
{
    public class Post : BaseEntity
    {        
        public Account Account{ get;set; }
        public int AccountId { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public DateTime CreatedDate { get; set; }
        public bool IsDeleted { get; set; }
         public List<Comment> Comments { get; set; }
         public List<Like> Likes { get; set; }
    }
    //DateTime.Now.ToString("dddd dd MMMM yyyy")
}